website/docs/components/form/super-select/partials/code/how-to-use.md

```diff
@@ -1,24 +1,21 @@
 ## How to use this component
 
 Super Select is a custom select-like component aiming to overcome some limitations of the HTML `<select>` element.
-It is based on the popular Ember [PowerSelect](https://ember-power-select.com/) add-on.
 
-We provide two main components with similar APIs, `SuperSelect::Single` and `SuperSelect::Multiple`.
+It's a light wrapper for Ember [PowerSelect](https://ember-power-select.com/), with some specific accessibility and styling choices made to best fit our design system.
+
+We provide two main components with similar APIs: `SuperSelect::Single` and `SuperSelect::Multiple`.
 
 **There are two ways to use each of the Super Select component types:**
 
-1. `Form::SuperSelect::Single::Base` or `Form::SuperSelect::Multiple::Base`—the base component with just the Super Select control.
+1. `Form::SuperSelect::Single::Base` or `Form::SuperSelect::Multiple::Base`—the base component with just the Super Select control. In these instances, you will need to add your own label, helper text, and error messaging.
 2. `Form::SuperSelect::Single::Field` or `Form::SuperSelect::Multiple::Field`—the field parent component which includes the Super Select control, label, helper text, and error messaging (in a wrapping container).
 
-We recommend using the Field component as it provides built-in accessibility functionality. Use the Base component for custom layouts or special use cases not covered by the Field component.
-
-### Field components
-
-#### Basic invocations including search
+We recommend using the Field variation as it provides built-in accessibility functionality. Try to only use the Base variation for custom layouts or special use cases not otherwise covered.
 
-##### SuperSelect::Single::Field
+### Field: Single-select
 
-Use `SuperSelect::Single` if you want users to only select a single option.
+In cases where the HDS Dropdown or HDS Form Select components are not suitable, use `SuperSelect::Single`. In this example, search is enabled:
```

Comment:
This is meant to represent a basic typical example and also to clarify the main difference between the Single and Multiple versions. This edit removes the explanation that it only allows users to select a single option. I also don't think it's necessary to say that search is enabled here when it's already mentioned in the header and can be seen in the example. (I set searchEnabled to true in the basic invocation examples since that seemed like the most common basic use case for SuperSelect that differentiates it from the Dropdown or Form::Select)

Maybe you could add another short paragraph above to briefly explain when to use SuperSelect vs. the Dropdown or Form::Select instead of here?

Reply:
For now I've removed "In this example" 
We can def work on the differences unless that's already in the overview?